### PR TITLE
Implement eth_sendRawTransaction for large Ethereum TXs

### PR DESCRIPTION
**Pull request by natanasow**
Signed-off-by: nikolay <n.atanasow94@gmail.com>

**Description**:
Support `eth_sendRawTransaction` for large Ethereum TXs

**Related issue(s)**:

Fixes https://github.com/hashgraph/hedera-json-rpc-relay/issues/15

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: packages/relay/src/lib/clients/sdkClient.ts [hunk 1]
```diff
@@ -184,6 +184,11 @@ export class SDKClient {
     }
 
     async submitEthereumTransaction(transactionBuffer: Uint8Array): Promise<TransactionResponse> {
+        if (transactionBuffer.length > 4096) {
```
Comment (Nana-EC) [inline packages/relay/src/lib/clients/sdkClient.ts:187]: Q: doesn't the sdk make this check for you?
If it's possible to just call the `EthereumFlow` always let's do that

Comment (Nana-EC) [inline packages/relay/src/lib/clients/sdkClient.ts:187]: I'm presuming an acceptance test case for a larger transaction will verify the support.

**Review by Nana-EC — COMMENTED** [review]
Looking good.
Early comments

File: packages/relay/src/lib/clients/sdkClient.ts (hunk 1 above)
Comment (ar-conmit) [inline packages/relay/src/lib/clients/sdkClient.ts:187]: nit: the transaction size can be moved to a constant var

**Comment by natanasow** [comment]
Blocked, waiting for [PR](https://github.com/hashgraph/hedera-sdk-js/pull/1182) to be merged.

File: packages/relay/src/lib/clients/sdkClient.ts (hunk 1 above)
Comment (natanasow) [reply packages/relay/src/lib/clients/sdkClient.ts]: The test is ready. Executed it locally with the updated version of the SDK and it passes, so once the [PR](https://github.com/hashgraph/hedera-sdk-js/pull/1182) is merged and released, we have to bump the SDK version in a relay and all should be good.

**Review by ar-conmit — DISMISSED** [review]
LGTM

File: packages/server/tests/acceptance/rpc.spec.ts [hunk 2]
```diff
@@ -354,6 +354,26 @@ describe('RPC Server Acceptance Tests', function () {
...
+                        gasLimit: defaultGasLimit,
+                        data: '0x' + '00'.repeat(5121),
+                    };
+
+                    const signedTx = await accounts[2].wallet.signTransaction(transaction);
+                    const transactionHash = await relay.call('eth_sendRawTransaction', [signedTx]);
+                    const info = await mirrorNode.get(`/contracts/results/${transactionHash}`);
+                    expect(info).to.have.property('created_contract_ids');
+                    expect(info).to.have.property('contract_id');
+                    expect(info.created_contract_ids.length).to.be.greaterThan(0);
```
Comment (Nana-EC) [inline packages/server/tests/acceptance/rpc.spec.ts:373]: Presuming it will only contain the actual created contract then you can make this equal 1

**Review by Nana-EC — COMMENTED** [review]
LG, also waiting on the skd-js pull to go through

**Review by Nana-EC — DISMISSED** [review]
LG

**Review by Nana-EC — DISMISSED** [review]
LGTM

**Review by Nana-EC — APPROVED** [review]

**Pull request merged**